lambdatimes/src/components/Content/Card.js

```diff
@@ -1,19 +1,27 @@
 import React from 'react';
+import PropTypes from 'prop-types'
 
 const Card = props => {
   return (
     <div className="card">
-      <div className="headline">{/* headline goes here */}</div>
+      <div className="headline">{props.card.headline}</div>
       <div className="author">
         <div className="img-container">
-          <img src={'' /* image source goes here */} />
+          <img src={props.card.img} />
         </div>
-        <span>By {/* author goes here */}</span>
+        <span>By {props.card.author}</span>
       </div>
     </div>
   );
 };
 
 // Make sure to include PropTypes.
+Card.propTypes = {
+  card: PropTypes.shape({
+    headline: PropTypes.string,
+    img: PropTypes.string,
+    author: PropTypes.author,
```

Comment:
With PropTypes.element you can specify that only a single child can be passed to a component as children. You just need to add a '.isRequired' after each propType. This is how it was done in the solution. Just letting you know!